ucsb_ds_capstone_projects_2021/projects/ccber/description.md

```diff
@@ -1,2 +1,9 @@
 # ccber
-Project description
+![](https://github.com/angelchen7/ucsb-ds-capstone-2021.github.io/blob/main/ucsb_ds_capstone_projects_2021/projects/ccber/CCBER_logo.jpg?raw=true)
```

Comment:
Markdown can embed images directly from the same directory: https://jupyterbook.org/content/figures.html
For the image  `CCBER_logo.jpg` in the same directory (you already have it), replace the HTML with `![some-text](CCBER_logo.jpg)`